new/LIT6939GadlaSosenna.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6939GadlaSosenna" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ገድለ፡ ሶስና፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Gadla Sosǝnnā</title>
+                <title xml:lang="en" corresp="#t1">Life of Susanna</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
```

Comment:
add MS Tel-Aviv, Faitlovich 8 as a witness
(record for Tel Aviv https://en-cenlib.tau.ac.il/special-collection-Faitlovich as INS and a stub for the MS should be created)

Reply:
We have INS0866TSL for Tel Aviv University Sourasky Central Library. If I understand the information in your link correct, the manuscript Faitlovich 8 is part of another collection with the name "Faïtlovitch Collection. Ancient Ethiopic Manuscripts Collection" and the first link was about a temporary exhibition of this collection in INS0866TSL.

Reply:
No. Faitlovich collection is in Souraski library. I have created the stub.